pytorch/training/docker/1.9/py3/Dockerfile.cpu

```diff
@@ -0,0 +1,147 @@
+FROM ubuntu:20.04
+
+LABEL maintainer="Amazon AI"
+LABEL dlc_major_version="1"
+
+ARG PYTHON=python3
+# Add arguments to achieve the version, python and url
+ARG PYTHON_VERSION=3.8.5
+ARG OPEN_MPI_VERSION=4.0.1
+
+# The smdebug pipeline relies for following format to perform string replace and trigger DLC pipeline for validating
+# the nightly builds. Therefore, while updating the smdebug version, please ensure that the format is not disturbed.
+ARG SMDEBUG_VERSION=1.0.9
+
+ARG DEBIAN_FRONTEND=noninteractive
+
+# Python won’t try to write .pyc or .pyo files on the import of source modules
+# Force stdin, stdout and stderr to be totally unbuffered. Good for logging
+ENV PYTHONDONTWRITEBYTECODE=1
+ENV PYTHONUNBUFFERED=1
+ENV PYTHONIOENCODING=UTF-8
+ENV LANG=C.UTF-8
+ENV LC_ALL=C.UTF-8
+ENV LD_LIBRARY_PATH="${LD_LIBRARY_PATH}:/usr/local/lib"
+ENV LD_LIBRARY_PATH="${LD_LIBRARY_PATH}:/opt/conda/lib"
+ENV PATH=/opt/conda/bin:$PATH
+ENV SAGEMAKER_TRAINING_MODULE=sagemaker_pytorch_container.training:main
+ENV DGLBACKEND=pytorch
+ENV TZ=America/New_York
```

Comment:
Configured default region for TZ data package.

Reply:
We are upgrading ubuntu version to 20.04 for PT 1.9 DLC, for PT 1.8 it was ubuntu 18. I followed solution suggested [here](https://serverfault.com/questions/949991/how-to-install-tzdata-on-a-ubuntu-docker-image) . Checking if we can just get away with only `ENV DEBIAN_FRONTEND=noninteractive`

Reply:
Removed the region configuration. Resolving the comment.